spk/flexget/Makefile

```
@@ -1,35 +1,36 @@
 SPK_NAME = flexget
-SPK_VERS = 2.9.14
-SPK_REV = 6
-SPK_ICON = src/flexget.png
+SPK_VERS = 3.1.79
+SPK_REV = 7
+SPK_ICON = src/${SPK_NAME}.png
 
+BUILD_DEPENDS = cross/python3 cross/setuptools cross/pip cross/wheel
 WHEELS = src/requirements.txt
-SPK_DEPENDS = "python>=2.7.8-11"
+SPK_DEPENDS = "python3>=3.7.7"
 
-MAINTAINER = Diaoul
+MAINTAINER = SynoCommunity
```

I propose you to take its ownership
```suggestion
MAINTAINER = manowark
```